src/OpenTelemetry.Exporter.OneCollector/Internal/EventNameManager.cs

```diff
@@ -112,7 +124,7 @@ public ResolvedEventFullName ResolveEventFullName(
         return resolvedEventFullName;
     }
 
-    private static byte[] BuildEventFullName(string eventNamespace, string eventName)
+    private static byte[] BuildEventFullName(string eventNamespace, string eventName, EventNameDelimiter eventNameDelimiter = EventNameDelimiter.Period)
```

Comment:
Probably by the time this function is reached, the eventNameDelimiter should be a byte and not an enum

Reply:
Actually, this shouldn't be a parameter - you should just reference the property of the EventNameManager

Reply:
Agreed.